### PR TITLE
adding some new tags that will definitely work

### PR DESCRIPTION
**Pull request by ckwolff**
Some new stuff for you max, it will definitely work, no need for a code review!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,6 @@
 Hello from CAs Wolrd!
+<!jflkjfdass>
```
Comment (maxwilets) [inline index.html:2]: ????????

File: index.html [hunk 2]
```diff
@@ -1 +1,6 @@
...
+<head>
```
Comment (maxwilets) [inline index.html:3]: closing tag?

File: index.html [hunk 3]
```diff
@@ -1 +1,6 @@
...
+<header>
+</stuff>
```
Comment (maxwilets) [inline index.html:5]: stuff isnt a tag?

File: index.html [hunk 4]
```diff
@@ -1 +1,6 @@
...
+</done>
```
Comment (maxwilets) [inline index.html:6]: ?????

**Comment by maxwilets** [comment]
@ckwolff needs some attention see comments

**Comment by ckwolff** [comment]
Whaaa? this will totally work

**Pull request merged**